cmd/internal/rssshrinker/rssshrinker_linux.go

```diff
@@ -0,0 +1,109 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux
+
+// Package rssshrinker provides functions to reduce the process’ RSS
+package rssshrinker
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"runtime/debug"
+	"strconv"
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
+/*
+#include <malloc.h>
+*/
+import "C"
+
+//revive:disable:var-naming
+
+// MADV_PAGEOUT Reclaim these pages.
+const MADV_PAGEOUT = 21
+
+//revive:enable:var-naming
+
+// Setup releases memory to the OS
+func Setup() {
+	// Release the memory garbage collected by the Go runtime to Linux
+	debug.FreeOSMemory()
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏This is not Linux specific so we could have a version of `rssshrinker` just calling it for Windows/Darwin